x-pack/test_serverless/functional/test_suites/common/discover/group3/_request_counts.ts

```diff
@@ -181,7 +181,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         setQuery: (query) => queryBar.setQuery(query),
       });
 
-      it('should send 2 requests (documents + chart) when adding a filter', async () => {
+      it.skip('should send 2 requests (documents + chart) when adding a filter', async () => {
```

Comment:
why is it skipped?

Reply:
It keeps on fail randomly, so I've skipped for now as it looks flaky somehow.
I've already collected it in the tracked follow up list: https://github.com/elastic/kibana/issues/195355